service/http/test_http_service.py

```
@@ -0,0 +1,51 @@
+        mock_get.return_value = mock_response
+        service = HttpService(base_url="https://api.example.com", endpoint="test")
+
+        #when
+        result = service.get()
+
+        #then
+        self.assertEqual(result, {"key": "value"})
+        mock_get.assert_called_once_with("https://api.example.com/test")
+
+    @patch('requests.get')
+    def test_get_failure(self, mock_get):
```

Ruff

Missing return type annotation for public function `test_get_failure`

Add return type annotation: `None`

(ANN201)